### PR TITLE
Use websockets for broadcasting drawings

### LINKED ISSUE #12: Use WS for drawings and pings
Either setup new WS server or expose existing one created by `boardgame.io`.
Sending drawings and pings without changing game state should reduce `Invalid StateID` errors

### PR DESCRIPTION
**Pull request by Bombasarkadian**
Closes #12 

### Note:
This change uses a forked version of boardgame.io: https://github.com/Bombasarkadian/boardgame.io/tree/allow-invalid-state which is prepackaged into `vendor` directory

### Changes:
- Use socket.io broadcast instead of regular move for updating drawing (`broadcast` flag)
- Update drawing in realtime
- Execute ping client-side
- Allow ping to be processed even when state doesn't match (`unsafe` flag)

### CODE REVIEW

File: src/games/kalambury/DrawArea.js [hunk 1]
```diff
@@ -15,12 +15,12 @@ const DrawArea = ({ initialLines, remainingSeconds, onUpdate, onForfeit }) => {
     return () => document.removeEventListener("mouseup", handleMouseUp);
   }, []);
 
-  useEffect(() => {
+/*  useEffect(() => {
```
Comment (pastDexter) [inline src/games/kalambury/DrawArea.js]: Do not leave comments - remove if not needed

File: src/games/kalambury/DrawArea.js [hunk 2]
```diff
@@ -71,7 +72,7 @@ const DrawArea = ({ initialLines, remainingSeconds, onUpdate, onForfeit }) => {
     if (onUpdate) onUpdate(lines);
   };
 
-  useInterval(updateGameState, 500);
+  //useInterval(updateGameState, 500);
```
Comment (pastDexter) [inline src/games/kalambury/DrawArea.js]: same as above ☝️

File: src/games/kalambury/Kalambury.js [hunk 3]
```diff
@@ -79,8 +79,8 @@ function ChangePhrase(G, ctx) {
   SetNewPhrase(G, ctx);
 }
 
-function UpdateDrawing(G, _ctx, lines) {
-  G.drawing = lines;
+function UpdateDrawing(G, _ctx, lines, qwe = "TEST") {
```
Comment (pastDexter) [inline src/games/kalambury/Kalambury.js]: `qwe = "TEST"`?

File: src/games/kalambury/board/DrawingBoard.js [hunk 4]
```diff
@@ -1,13 +1,15 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
```
Comment (pastDexter) [inline src/games/kalambury/board/DrawingBoard.js]: nothing really changed here

you are not using `rawClient`

File: README.md [hunk 5]
```diff
@@ -2,15 +2,18 @@
...
 Install dependencies `npm install`.
 
 Run react dev server with `npm start`.
 
 Run game server with: `node -r esm src/server.js`.
 
-Setting `debug=true` in URL query will show debug overlay. 
+Setting `debug=true` in URL query will show debug overlay.
+
+Run `npm run prettier` before commiting your changes 🙌
```
Comment (pastDexter) [inline README.md]: ...or use IDE that does it for you on every save :P

File: src/contexts/UserContext.js [hunk 6]
```diff
@@ -1,46 +1,44 @@
-import React, { createContext, useState, useMemo, useCallback } from 'react';
+import React, { createContext, useState, useMemo, useCallback } from "react";
 
-import { UserClient } from '../services/User';
+import { UserClient } from "../services/User";
```
Comment (pastDexter) [inline src/contexts/UserContext.js]: use absolute imports `services/User`

**Comment by pastDexter** [comment]
works like a charm when deployed to heroku 🎉:clap:

File: src/contexts/UserContext.js (hunk 6 above)
Comment (Bombasarkadian) [reply src/contexts/UserContext.js]: There are more imports like this one. We should do a project-wide cleanup

**Pull request merged**